backend/backend/start_game.py

```diff
@@ -28,3 +30,9 @@ def start_game_db(game_id):
     """Changes a game’s status to 'playing' in the database."""
     with backend.game.Game(game_id) as game:
         game.state = 'playing'
+        players = game.players
+        shuffle(players)
+        for x, player in enumerate(len(players)):
```

Comment:
Is there meant to be a call to `len()` here? I might not be following, but I think `enumerate()` is meant to be called directly on a sequence, rather than on its length.